tests/expected_output/thiserror-1.0.30.txt

```diff
@@ -1,2 +1,2 @@
 pub mod thiserror
-pub proc macro thiserror::Error!
+pub proc macro thiserror::#[derive(Error)]
```

Comment:
Sorry, I created a conflict with you here with https://github.com/Enselic/public_items/pull/68

I try to not create conflicts but forgot to double-check this one

Fortunately it should be easy for you to resolve; change from
```
pub proc macro comprehensive_api_proc_macro::SimpleDeriveMacro!
```
to
```
pub proc macro comprehensive_api_proc_macro::#[derive(SimpleDeriveMacro)]
```
in ./tests/expected_output/comprehensive_api_proc_macro.txt

Reply:
> (The reason I work towards removing all .json from the repo is that that makes it easier to rename the lib to public-api later.)

Just a note (also mentioned in a separate comment): I now feel done with a big restructuring on how tests are run. Mainly stopping to use public_items itself for tests, because that becomes a real pain when we will rename the lib to public-api. The PR is at https://github.com/Enselic/public_items/pull/70, but I will not merge it before this PR is merged, to avoid causing more troubles for you. Again, I am very open to resolving the thiserror-1.0.30.txt-conflict for you, if you wish. It is not fair for you to solve problems caused by me :)